JetStreamDriver.js

```diff
@@ -2371,108 +2374,22 @@ function processTestList(testList)
     else
         benchmarkNames = testList.split(/[\s,]/);
 
-    const forceEnable = true;
     for (const name of benchmarkNames) {
         if (benchmarksByTag.has(name))
-            enableBenchmarksByTag(name, forceEnable);
+            enableBenchmarksByTag(name);
         else
             enableBenchmarksByName(name);
     }
 }
 
-let runOctane = true;
-let runARES = true;
-let runWSL = true;
-let runRexBench = true;
-let runWTB = true;
-let runSunSpider = true;
-let runBigIntNoble = true;
-let runBigIntMisc = true;
-let runProxy = true;
-let runClassFields = true;
-let runGenerators = true;
-let runSimple = true;
-let runCDJS = true;
-let runWorkerTests = !!isInBrowser;
-let runSeaMonster = true;
-let runCodeLoad = true;
-let runWasm = true;
-if (typeof WebAssembly === "undefined")
-    runWasm = false;
-
-if (false) {
-    runOctane = false;
-    runARES = false;
-    runWSL = false;
-    runRexBench = false;
-    runWTB = false;
-    runSunSpider = false;
-    runBigIntNoble = false;
-    runBigIntMisc = false;
-    runProxy = false;
-    runClassFields = false;
-    runGenerators = false;
-    runSimple = false;
-    runCDJS = false;
-    runWorkerTests = false;
-    runSeaMonster = false;
-    runCodeLoad = false;
-    runWasm = false;
-}
+const defaultDisabledTags = [];
+if (!isInBrowser)
```

Comment:
Nit: I would just check for `globalThis.Worker`. If a CLI adds `Worker` support I assume they'd want these to run.